tasks/definitions/index.js

```diff
@@ -1,10 +1,10 @@
-const { isObj, isFunc, mapObj } = require('@keg-hub/jsutils')
-const { getHerkinConfig } = require('../../configs/herkin.config')
+const { isObj, isFunc, mapObj, pipeline } = require('@keg-hub/jsutils')
```

Comment:
Doesn't look like `pipeline` is being used anywhere?

Reply:
removed